paimon-web-ui/src/api/models/catalog/types/table.ts

```diff
@@ -67,11 +74,13 @@ export interface TableDetail {
 }
 
 export interface ColumnDTO {
+  id: number
```

Comment:
Id is not required when creating a new column.

Reply:
Yes, you don't need to pass the id when adding a column, but you need to pass the id and sort when editing a column so that the backend can determine the new order of each field. Do I need to create a new user to edit it instead of sharing the same one with the creation?